tutorials/scripting/signals_concise_example.rst

```diff
@@ -0,0 +1,146 @@
+.. _doc_signals_concise_example:
+
+Signals, concise example
+========================
+
+Signals are easy.
```

Comment:
I know what you mean, but this is more an opinion than anything else. I'm more in the mindset of showing how easy it is instead of telling how easy it is.